addons/website/static/src/js/recaptcha.js

```diff
@@ -0,0 +1,46 @@
+odoo.define('reCaptchaV3Mixin', function (require) {
+"use strict";
+
+/**
+ * The ReCaptchaV3Mixin is a mixin, designed to add the google reCaptchaV3
+ */
+return {
```

Comment:
cant we use the explicit rendering method and use inline mode to make it invisible (https://developers.google.com/recaptcha/docs/invisible)

or in worst case: hide in css the badge:
```
.grecaptcha-badge {
    display: none !important;
}
```

Reply:
https://developers.google.com/recaptcha/docs/faq#id-like-to-hide-the-recaptcha-badge.-what-is-allowed

A branding is still required...